### PR TITLE
11723 - Filter Styles

### PR DESCRIPTION
**Pull request by andreferreiradlw**
[TP11723](https://maps.tpondemand.com/entity/11723-tad-filter-section-styling)

In this PR, some adjustments were made to the styles of certain components in order to bring them closer to the existing styles on RAD. The components modified are `Tooltip`, `FormField` and `Radio`, adding a new prop to `FormField` that allows the consumer app to change the color of the legend if needed.

Also updated the plugins for `Rollup` and managed some dependencies in `package.json`.

### CODE REVIEW

File: src/components/Radio/StyledRadio.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const Label = styled.label`
 `
 
 const Field = styled.input`
-  margin-right: 5px;
+  margin-right: 10px;
```
Comment (davidtrussler) [inline src/components/Radio/StyledRadio.js:11]: I think this change is fine. On RAD we gave added an explicit value of zero to the other margins to reset any browser-set values. I'm wondering if we should do that for this as well? Or would it be better to set that in the app?

File: src/components/Tooltip/StyledTooltip.js [hunk 2]
```diff
@@ -77,14 +77,24 @@ const Tip = styled.span`
...
 `
 
 const Icon = styled(Anchor)`
+  font-family: 'Times-Italic', 'Times Italic', 'Times', sans-serif;
   font-style: italic;
-  padding: 0.375rem;
+  font-weight: 400;
+  color: ${({ theme }) => theme.colors.tooltip.iconColor};
+  margin: 0;
+  border: 1px solid black;
```
Comment (davidtrussler) [inline src/components/Tooltip/StyledTooltip.js:91]: We should use the MAS green for the border of the icon.

Comment (ladamsatmaps19) [reply src/components/Tooltip/StyledTooltip.js:91]: well spotted @davidtrussler. Let us know when you've made this small change @andreferreiradlw :)

File: src/components/Radio/StyledRadio.js (hunk 1 above)
Comment (andreferreiradlw) [reply src/components/Radio/StyledRadio.js:12]: Initially I didn't want to give it any margin either but Nakul specifically asked for it to match the wireframes so I basically doubled it here.

File: src/components/Tooltip/StyledTooltip.js (hunk 2 above)
Comment (andreferreiradlw) [reply src/components/Tooltip/StyledTooltip.js]: I've adjusted this now to MAS green on the theme and made this dynamic

File: src/components/Radio/StyledRadio.js (hunk 1 above)
Comment (davidtrussler) [reply src/components/Radio/StyledRadio.js:12]: > Initially I didn't want to give it any margin either but Nakul specifically asked for it to match the wireframes so I basically doubled it here.

OK, no problem, though there seems to have been a little confusion throughout this project on whether the wireframes or existing RAD/MAS styles are being followed.

**Review by davidtrussler — APPROVED** [review]
LGTM! 👍

**Review by akenzua — APPROVED** [review]
LGTM

**Pull request merged**